src/optimize-initialize.py

```
@@ -0,0 +1,133 @@
+        args = shlex.split(command_line)
+
+        print(args)
+
+        output = qx(args)
+
+        # Parse output into a list of [simplices, min timeslice, max timeslice]
+        result = [0, 0, 0]
+        graph = []
+        for line in output.splitlines():
+            if line.startswith("Minimum timevalue"):
+                s = re.findall('\d+', line)
```

Ruff (0.15.21)

[warning] Invalid escape sequence: `\d`

Use a raw string literal

(W605)